example/basic_example.py

```diff
@@ -11,23 +11,47 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
+import random
 import time
 
 from dynatrace.opentelemetry.metrics.export import DynatraceMetricsExporter
 from opentelemetry import metrics
 from opentelemetry.sdk.metrics import MeterProvider
 from os.path import splitext, basename
 import argparse
+import logging
+import os
+import psutil
+
+
+def get_random_number(maximum: int, minimum: int = 0):
+    if maximum < minimum:
+        minimum, maximum = maximum, minimum
+
+    return random.randint(minimum, maximum)
```

Comment:
Since you're just using it four times here (and always with the correct params 😉), you could also just call `random.randint(0, x)` or `random.randrange(x+1)` directly.